bash_functions.sh

```diff
@@ -27,12 +27,13 @@ prepare_configs() {
     chown pihole:root /etc/lighttpd
     chown pihole:pihole "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" "/var/log/pihole" "${regexFile}"
```

Comment:
Consequently `"${regexFile}"` can then be removed above as well, to mute the last error message.

Reply:
Thanks, well spotted. Deleting with half an eye...

Reply:
Except now I have to wait for the rebuild again 🙄